Clicker/SectionControllers/SettingsSectionController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  SettingsSectionController.swift
+//  Clicker
+//
+//  Created by eoin on 9/13/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import IGListKit
+
+class SettingsSectionController: ListSectionController {
+    
+    var settingsDataModel: SettingsDataModel!
+    
+    // MARK: - ListSectionController overrides
+    override func sizeForItem(at index: Int) -> CGSize {
+        guard let containerSize = collectionContext?.containerSize else {
+            return .zero
+        }
+        switch settingsDataModel.state {
+        case .info:
+            return CGSize(width: containerSize.width, height: settingsDataModel.title == "About" ? 110 : 87)
+        case .link:
+            return CGSize(width: containerSize.width, height: 39.5)
+        }
+    }
```

Comment:
make each of the constants in this function a class variable, i.e.:
`let cellHeight = 110`